### PR TITLE
Update of 302 and 305 NNCF notebooks. Logging and metrics.

### PR DESCRIPTION
**Pull request by negvet**
- Remove logging of INFO and WARNING levels (warning may confuse users, basically all actions are explicitly commented in the appropriate cells). 
- Update output metrics in TF to be aligned.

### CODE REVIEW

**Review by ljaljushkin — APPROVED** [review]

**Comment by helena-intel** [comment]
Thanks @negvet it is very welcome to have fewer warning messages about things that are not actually a concern. I prefer not to blanket hide all warnings though - presumably these warnings are there for a reason? I'm also not against hiding all NNCF warnings in the notebook if the alternatives are uglier, but then we should have a comment above that that explains that NNCF displays some warnings that can be ignored for the purposes of the tutorial - that hopefully helps with people who blindly copy/paste all the imports.

It also seems not ideal to hide all warnings - maybe other libraries do show warnings that we want to see. But I still see warnings with this `warnings.disable` call. Is that expected?
image

**Comment by negvet** [comment]
@helena-intel thanks for the feedback.
I updated the notebooks so that only NNCF-related logging is disabled. Note is also added.
Please check it out.

File: notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb [hunk 1]
```diff
@@ -35,7 +35,9 @@
     "On Windows, add the required C++ directories to the system's path.\n",
     "\n",
     "Import NNCF and all auxiliary packages from your Python code.\n",
-    "Set a name for the model, and the image width and height that will be used for the network. Also define paths where PyTorch, ONNX and OpenVINO IR versions of the models will be stored. "
+    "Set a name for the model, and the image width and height that will be used for the network. Also define paths where PyTorch, ONNX and OpenVINO IR versions of the models will be stored. \n",
+    "\n",
+    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disbaled to simplify the tutorial. For real usage scenario it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
```
Comment (helena-intel) [inline notebooks/302-pytorch-quantization-aware-training/302-pytorch-quantization-aware-training.ipynb:40]:
```suggestion
    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disabled to simplify the tutorial. For production use, it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
```

File: notebooks/305-tensorflow-quantization-aware-training/305-tensorflow-quantization-aware-training.ipynb [hunk 2]
```diff
@@ -24,9 +24,10 @@
    "metadata": {},
    "source": [
     "## Imports and Settings\n",
-    "\n",
     "Import NNCF and all auxiliary packages from your Python code.\n",
-    "Set a name for the model, input image size, used batch size, and the learning rate. Also define paths where Frozen Graph and OpenVINO IR versions of the models will be stored."
+    "Set a name for the model, input image size, used batch size, and the learning rate. Also define paths where Frozen Graph and OpenVINO IR versions of the models will be stored.\n",
+    "\n",
+    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disbaled to simplify the tutorial. For real usage scenario it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
```
Comment (helena-intel) [inline notebooks/305-tensorflow-quantization-aware-training/305-tensorflow-quantization-aware-training.ipynb:30]:
```suggestion
    "> NOTE: All NNCF logging messages below ERROR level (INFO and WARNING) are disabled to simplify the tutorial. For production use, it is recommended to enable logging, by removing ```set_log_level(logging.ERROR)```."
```

**Review by helena-intel — CHANGES_REQUESTED** [review]
Thanks! I have a minor language suggestion, otherwise it looks good. I still see this warning: 
image
I suppose this is because of warnings instead of logging, and the note is clear about that. Are we missing important NNCF functionality by using PyTorch 1.7? I would like to update to a higher version of PyTorch, but the 102 notebook currently breaks with a higher version.

**Comment by negvet** [comment]
> Are we missing important NNCF functionality by using PyTorch 1.7?

@ljaljushkin might you please advise here?

**Comment by ljaljushkin** [comment]
> > Are we missing important NNCF functionality by using PyTorch 1.7?
> 
> @ljaljushkin might you please advise here?

AFAIK, we don't test nncf with all previous versions of pytorch, that's why we don't guarantee best results with them.
nncf is validated with torch1.9.1.

**Comment by negvet** [comment]
Jenkins please retry a build

**Review by helena-intel — APPROVED** [review]

**Comment by helena-intel** [comment]
Thanks @negvet !

**Pull request merged**